packages/ui/src/components/va-input/VaInput.vue

```diff
@@ -137,6 +137,7 @@ export default defineComponent({
       ...validationListeners,
       class: attrs.inputClass,
       style: attrs.inputStyle,
+      id: attrs.inputId,
```

Comment:
Better use id and wrapperId, because usually we want to set id to input tag instead of va-input root element.

Reply:
fixed: set all `attrs` (except `class` and `style`) to `input` tag. Only `class` and `style` `attrs` are set to `va-input`